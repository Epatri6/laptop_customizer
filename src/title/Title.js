import React from 'react';
import './Title.css';

export default class Title extends React.Component {

  render() {
    return (         
      <header>
        <h1>ELF Computing | Laptops</h1>
      </header>
    );
  }

}