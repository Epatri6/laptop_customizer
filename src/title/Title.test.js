import React from 'react';
import ReactDOM from 'react-dom';
import Title from './Title';
import renderer from 'react-test-renderer';

describe('Testing Title Component', () => {

  //Smoke test
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Title />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  //Snapshot test
  it('renders as expected', () => {
    const tree = renderer
      .create(<Title />)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });
});