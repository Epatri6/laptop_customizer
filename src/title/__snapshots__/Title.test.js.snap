// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Title Component renders as expected 1`] = `
<header>
  <h1>
    ELF Computing | Laptops
  </h1>
</header>
`;
